docs/integrations/azure/locations.md

```diff
@@ -25,7 +37,17 @@ integrations:
           tenantId: ${AZURE_TENANT_ID}
 ```
 
-Using a managed identity:
+#### Using a system-assigned managed identity
+
+```yaml
+integrations:
+  azure:
+    - host: dev.azure.com
+      credentials:
+        - clientId: system-assigned
+```
+
+#### Using a user-assigned managed identity
```

Comment:
Same here: How does one know this? Can we link to some Microsoft docs on the subject?

Reply:
@awanlin I have updated the entire section, please have another look